typed-racket-lib/typed-racket/base-env/base-env.rkt

```diff
@@ -1085,6 +1089,10 @@
 [make-immutable-custom-hash (->opt (-> Univ Univ Univ) (-> Univ -Nat) [(-> Univ -Nat)] Univ)]
 [make-weak-custom-hash (->opt (-> Univ Univ Univ) (-> Univ -Nat) [(-> Univ -Nat)] Univ)]
 
+
+[hash-union (-poly (a b) (->* (list (-Immutable-HT a b))  (-HT a b) (-Immutable-HT a b)))]
+[hash-union! (-poly (a b) (->* (list (-Mutable-HT a b))  (-HT a b) -Void))]
```

Comment:
`hash-intersect` too?

Reply:
This was just aiming at #1025, which is just those two functions.

Reply:
In fact, there is no `hash-intersect`

Reply:
See https://github.com/racket/racket/pull/3310

Reply:
Ah, that's what I get for looking at docs.racket-lang.org. Unfortunately there's a typo in the docs ...